cranelift/codegen/src/isa/aarch64/lower.isle

```diff
@@ -1124,3 +1124,117 @@
 
 (rule (lower (has_type $I8X16 (popcnt x)))
       (vec_cnt x (VectorSize.Size8x16)))
+
+;;;; Rules for `fcmp` 32 bit ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(rule (lower (has_type ty @ (multi_lane _ _) (fcmp (fcmp_cond_notEq cond) x (def_inst (splat (def_inst (f32const (zero_value_f32 y))))))))
+      (let (
```

Comment:
It's a tiny stylistic nit but I'd prefer to avoid this paren on its own line style; the more common S-expr formatting is something like

```
(let ((x ...)
      (y ...)
      (z ...))
  body)
```